src/commands/test/lib/help-messages.ts

```diff
@@ -7,27 +7,27 @@ export module Messages {
       export const GenerateUITest = "Generates a Xamarin.UITest project";
 
       export const PrepareAppium = "Creates an artifacts directory with Appium tests.";
-      export const PrepareCalabash = `Creates an artifacts directory with Calabash tests.${os.EOL}` + 
-                                     `Required external tools:${os.EOL}` + 
```

Comment:
Can we try to work out why your dev system keeps touching files like this when there are no changes? What are you using to edit files?

Reply:
Actually, it looks like we had some files previously checked in with trailing whitespace. This is a good thing - I really need to get TSLint hooked up to this thing.